jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java

```diff
@@ -1333,13 +1334,28 @@ public static DslViewResultsTree resultsTreeVisualizer() {
     return new DslViewResultsTree();
   }
 
+  /**
+   * Builds a ConstantThrouputTimer which paces samplers to achieve a target constant throughput.
+   *
+   * @param throughput specifies the target samples per minute.
+   * @param mode       specifies the calculation mode to apply.
+   * @return the timer for usage in test plan.
+   * @see DslConstantThroughputTimer
+   */
+
+  public static DslConstantThroughputTimer constantThroughputTimer(
```

Comment:
could we only require one parameter here (the throughput) and use a good default for mode allowing to modify it by chaining a method like .byThread? What is the most common scenario you use here? It sounds like all active threads shared would be the most common setting used. 
 Do you find by thread group and all active threads non shared useful for some scenarios? If you actually don't need them we may keep them out of current implementation and add them later if anyones actually need them to keep the API simple and properly document scenarios where each setting makes sense.